cmake/common.cmake

```diff
@@ -189,70 +199,206 @@ Unit Test Specification
 
 ")
 
-                # create the unit_test_results.rst file
-                set(_unit_test_results_rst ${SPHINX_OUTPUT_DIR}/unit_test_results.rst)
-                file(RELATIVE_PATH _rel_unit_test_results_rst ${SPHINX_SOURCE_DIR} ${_unit_test_results_rst})
+                # create the test results rst file
+                set(_unit_test_results_rst ${_component_reports_out_dir}/unit_test_results.rst)
                 file(WRITE ${_unit_test_results_rst} "
 Unit Test Results
 =================
 
 .. test-report:: Unit Test Results
-    :id: TEST_RESULT
-    :file: {{ component_test_junit_xml }}
+    :id: TEST_RESULT_${component_name}
+    :file: ${_component_test_junit_xml}
+
+")
+
+                # create the code coverage rst file

```

Comment:
Create coverate rst file to be able to automatically link to the coverage/index.html